spec/spec.md

```diff
@@ -1353,14 +1406,17 @@ JSONPath                      | Description
 
 ### CHAPI
 
-The [credential handler api (CHAPI)](https://w3c-ccg.github.io/credential-handler-api/) allows a web page to request data from a browser, and for a wallet to fulfill that request. This is commonly used for requesting and presenting verifiable credentials.
+The [credential handler api (CHAPI)](https://w3c-ccg.github.io/credential-handler-api/)
+allows a web page to request data from a browser, and for a wallet to fulfill
+that request. This is commonly used for requesting and presenting verifiable
+credentials.
 
 See also the [vp-request-spec](https://digitalbazaar.github.io/vp-request-spec/).
 
 Here is an example of a request:
 
 ::: example Presentation Definition using CHAPI
-```json
+```jsonc
```

Comment:
If we want comments, we should use JSON5 (https://spec.json5.org/), as that is a formatting option I can update the tool to render. Should

Reply:
to add JSON5 blocks, you can just use the string `json5` for these fenced block tags.

Reply:
Actually, I just checked, and the renderer's plain `json` tag already supports comments, so you can leave that regular tag and it will render correctly.

Reply:
fixed in [fa617c3](https://github.com/decentralized-identity/presentation-exchange/pull/79/commits/fa617c300055bbf5b19c0edcf98d1fbac38b416b)